docs/configuration/prune.rst

```diff
@@ -165,3 +159,55 @@ Like all other regular expression fields in prune policies, zrepl uses Go's `reg
 The optional `negate` boolean field inverts the semantics: Use it if you want to keep all snapshots that *do not* match the given regex.
 
 
+Source-side snapshot pruning
+----------------------------
+
+A :ref:`source jobs<job-source>` takes snapshots on the system it runs on.
+The corresponding :ref:`pull job <job-pull>` on the replication target connects to the source job and replicates the snapshots.
+Afterwards, the pull job coordinates pruning on both sender (the source job side) and receiver (the pull job side).
+
+There is no built-in way to define and execute pruning on the source side independently of the pull side.
+The source job will continue taking snapshots which will not be pruned until the pull side connects.
+This means that **extended replication downtime will fill up the source's zpool with snapshots**.
+
+If the above is a conceivable situation for you, consider using :ref:`push mode <job-push>`, where pruning happens on the same side where snapshots are taken.
+
+Workaround using ``snap`` job
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As a workaround (see GitHub :issue:`102` for development progress), a pruning-only :ref:`snap job <job-snap>` can be defined on the source side:
+The snap job is in charge of snapshot creation & destruction, whereas the source job's role is reduced to just serving snapshots.
+However, since, jobs are run independently, it is possible that the snap job will prune snapshots that are queued for replication / destruction by the remote pull job that connects to the source job.
+There is no possible configuration to avoid this problem with the current version of zrepl.
+
+Example configuration:
```

Comment:
I expaned the initially-provided example configuration in 3543fbbb650fb0e09a73368232d6e18b8734437f
However, I am unsure whether we are still talking the same approach.
@InsanePrawn  Could you please verify that the example configuration covers your use-case?